<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formular_adaugare_job</class>
 <widget class="QDialog" name="formular_adaugare_job">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 242, 241);</string>
  </property>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>320</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 23px;</string>
   </property>
   <property name="text">
    <string>Adaugă un loc de muncă</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>190</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 21px;</string>
   </property>
   <property name="text">
    <string>Domeniu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_domeniu">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>130</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_salvare">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>630</y>
     <width>140</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(9, 52, 111);
color: white;
font-size: 21px;
border-style: outset;
border-radius: 20px;
font: Times New Roman;</string>
   </property>
   <property name="text">
    <string>Salvare</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_functie">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>220</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_functie">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>190</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 21px;</string>
   </property>
   <property name="text">
    <string>Funcție</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_angajator">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>310</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_angajator">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>190</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 21px;</string>
   </property>
   <property name="text">
    <string>Angajator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_judet">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>400</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_judet">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>190</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 21px;</string>
   </property>
   <property name="text">
    <string>Județ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oras">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>490</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_oras">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>190</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 21px;</string>
   </property>
   <property name="text">
    <string>Oraș</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_program">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>580</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_program">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>190</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 175, 169);
color: white;
border-radius: 10px;
font: Times New Roman;
font-size: 21px;</string>
   </property>
   <property name="text">
    <string>Program de lucru</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_program_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>New Roman</family>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../PROIECT/logo_mic.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
